plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/views/featuremodeleditview/ExampleParent.java

```diff
@@ -78,8 +78,6 @@ public void initChildren() {
 		try {
 			if (example == null) {
 				example = c.calculateExample(added);
-			}
-			if (example == null) {
 				addChild(NONE);
```

Comment:
Be careful! This change is not semantics preserving! Please revert those changes unless you have clear evidence that it improves the behavior.